includes/Core/Util/Activation.php

```diff
@@ -122,12 +122,13 @@ function( $hook_suffix ) {
 		);
 		add_action(
 			'admin_enqueue_scripts',
-			function() {
-				if ( ! $this->get_activation_flag( is_network_admin() ) ) {
+			function( $hook_suffix ) {
+				if ( 'plugins.php' !== $hook_suffix || ! $this->get_activation_flag( is_network_admin() ) ) {
```

Comment:
Possibly. I personally think using the general hook here plus the condition keeps this more clear to read, and easier to update. The hook change would otherwise apply to a few other locations (e.g. right above this) as well, and hook names would potentially end up becoming very long as a result.